app/src/main/java/com/portfolio/tasky/usecases/NetworkChangeReceiver.kt

```diff
@@ -0,0 +1,67 @@
+package com.portfolio.tasky.usecases
```

Comment:
This is not a use case, since it interacts with an API (the Android NetworkManager API). This should be in your data layer

Reply:
Thanks for the info. Created a package data and shifted it there.